zhaquirks/thirdreality/switch_v2.py

```diff
@@ -0,0 +1,59 @@
+"""Third Reality Blind Gen2 devices."""
```

Comment:
Is this correct? Or should it be something like this?
```suggestion
"""Third Reality Switch Gen2 devices."""
```